receipt_langsmith/tests/spark/test_evaluator_diff_viz_cache.py

```
@@ -0,0 +1,174 @@
+    # Print transition summary
+    transitions: Counter[tuple[str | None, str]] = Counter()
+    source_counts: Counter[str] = Counter()
+    for receipt in diff_cache:
+        for w in receipt["words"]:
+            if w["changed"]:
+                before = w["before_label"] or "null"
+                after = w["after_label"]
+                transitions[(before, after)] += 1
+                source_counts[w["change_source"]] += 1
+
+    print(f"\n=== Label Transition Summary ===")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)